submodules/faxbox/faxbox.js

```diff
@@ -221,6 +230,25 @@ define(function(require) {
 					} else {
 						callback(null, {});
 					}
+				},
+				phone_numbers: function(callback) {
+					self.callApi({
+						resource: 'numbers.listAll',
+						data: {
+							accountId: self.accountId
+						},
+						success: function(_data) {
+							_data.numbers = _.chain(_data)
+								.get('data.numbers', {})
+								.map(function(value, number) {
+									return { label: number };
```

Comment:
I'd rather have kept the `id` prop instead of `label` as it more descriptive of what value it holds.
```suggestion
									return { id: number };
```
Since we only care about the keys of `numbers` object, another more elegant solution would be to use `lodash#keys` instead of a `lodash#map` to return a simple array of numbers.